src/data_vis.ts

```diff
@@ -192,6 +194,10 @@ class DataVisModel {
         };
     }, this).extend({rateLimit: 250});
 
+    private metaData = ko.computed<string>(() => {
```

Comment:
This updates nicely when I change the outcome for the plot (deaths to dalys) but when I reduce the number of bars so I only see Pakistan, Ethiopia and Nigeria in the plot, the metadata still says 'Countries: India, Pakistan, Nigeria, Ethiopia'. Is that expected?
![screenshot from 2019-03-08 10-26-10](https://user-images.githubusercontent.com/44669576/54023366-b33f2e80-418c-11e9-9a66-d782b9c398ba.png)

Reply:
This is difficult; the user has set the sidebar filters to include four countries. but then asked to only only show the largest three.

it's not clear if we should list all of the countries that were selected but that only the largest three are shown. Or only list the largest three.

It is easier to link the metadata to what the user asked for (first option) than to the filtered data  (second option). I'll implement the former, then if that still looks wrong I'll think about the latter.